File: yb/java/MainActivity.java
package com.example.orangeline;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {
    Button loginBtn;
    String UserId = "456";


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        loginBtn = findViewById(R.id.loginButton);

        loginBtn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(getApplicationContext(), RoomActivity.class);
                intent.putExtra("userId", UserId);
                startActivity(intent);
            }
        });
    }
}

File: cw/v2/OrangeLine/app/src/main/java/com/example/orangeline/MainActivity.kt
package com.example.orangeline

import java.util.*
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import com.kakao.sdk.common.util.Utility

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        var keyHash = Utility.getKeyHash(this)
        Log.i("keyhash: ", keyHash) // oTvsY0O2qpncZ/7FncqHRR0Jl5Q=

    }
}